scripts/json_verify.py

```diff
@@ -100,6 +100,13 @@ def find_uris(categories_json):
             for entity_name, entity_json in entity.iteritems():
                 assert type(entity_name) is UnicodeType
                 assert type(entity_json) is DictType
+                # pop dnt out of the dict, so we can iteritems() over the rest
+                try:
+                    dnt_value = entity_json.pop('dnt', '')
```

Comment:
Yes, `.pop('dnt', '')` returns the `dnt` item out of `entity_json` or defaults to `''`. So if the `dnt` item is missing, this returns an empty string.